pkg/services/sqlstore/apikey_test.go

```diff
@@ -1,31 +1,118 @@
 package sqlstore
 
 import (
-	"testing"
-
+	"github.com/grafana/grafana/pkg/models"
 	. "github.com/smartystreets/goconvey/convey"
-
-	m "github.com/grafana/grafana/pkg/models"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
 )
 
 func TestApiKeyDataAccess(t *testing.T) {
+	mockTimeNow()
+	defer resetTimeNow()
 
 	Convey("Testing API Key data access", t, func() {
 		InitTestDB(t)
 
 		Convey("Given saved api key", func() {
-			cmd := m.AddApiKeyCommand{OrgId: 1, Name: "hello", Key: "asd"}
+			cmd := models.AddApiKeyCommand{OrgId: 1, Name: "hello", Key: "asd"}
```

Comment:
Remove usage of convey in this file, see https://github.com/grafana/grafana/blob/master/pkg/README.md#reduce-the-use-of-goconvey